config/database.yml

```diff
@@ -14,4 +14,5 @@ test:
   database: <%= ENV.fetch('DATABASE_NAME', 'timeoverflow_test') %>
 
 production:
-  url: <%= ENV.fetch('DATABASE_URL', 'Set DATABASE_URL environment variable') %>
+  # Set DATABASE_URL environment variable
+  url: <%= ENV.fetch('DATABASE_URL', 'null') %>
```

Comment:
http://guides.rubyonrails.org/configuring.html#connection-preference
>If you have an empty config/database.yml file but your ENV['DATABASE_URL'] is present, then Rails will connect to the database via your environment variable

So that line could/should be removed